kernel/Makefile

```diff
@@ -34,7 +34,7 @@ WARNINGS += -Wno-conversion
 #USE_GC_SECTIONS=1
 
 CFLAGS =
-CFLAGS += -g
+CFLAGS += -g3
```

Comment:
This allows debugging into macros. I can probably remove it now things are working...

Reply:
I found my proposal would clean up the intermediate files no matter the tests passed or failed. But then the return code of "make clean" become the return code of the pipeline step, i.e., the pipeline reports **SUCCESS** even though one of the tests failed because the subsequent `make clean` is successful.

I withdraw my suggestion.